package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "anatine",
   "productName": "Anatine",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Never bump the version in a PR. It should be up to the maintainer to decide what version to bump, when to bump, and how to bump (I for example, use an automated script) ;)